src/components/tutor-input.cjsx

```diff
@@ -47,7 +47,11 @@ TutorTextArea = React.createClass
     onChange: React.PropTypes.func
     value: React.PropTypes.any
 
-  onChange: ->
+  resize: (event) ->
+    event.target.style.height = ''
```

Comment:
That's so we can get the scroll height.  Because of the material css, we need to clear the height first.  This should probably be 0px to be safe.